src/Symfony/Component/Form/Tests/FormConfigTest.php

```diff
@@ -21,56 +22,78 @@
  */
 class FormConfigTest extends TestCase
 {
-    public static function getHtml4Ids()
+    public static function provideInvalidFormInputName(): iterable
     {
-        return [
```

Comment:
can we keep the code style? That'll help with merges in the future and there's nothing wrong with the current way

Reply:
Done